core/themes/basis/template.php

```diff
@@ -25,6 +25,13 @@ function basis_preprocess_page(&$variables) {
   if ($view) {
     $variables['classes'][] = 'view-name-' . $view->name;
   }
+
+  // Add breakpoint-specific CSS for dropdown menus.
+  backdrop_add_css(backdrop_get_path('theme', 'basis') . '/css/component/menu-dropdown.breakpoint.css', array(
```

Comment:
Should we be using `path_to_theme()` instead of `backdrop_get_path()` here? I believe that it would work better when this template is copied to a subtheme:

```suggestion
  backdrop_add_css(path_to_theme() . '/css/component/menu-dropdown.breakpoint.css', array(
```

Reply:
@klonos Hmm, I'm not sure... `path_to_theme()` points to the current theme, right? So if someone copied Basis' `template.php` to their sub-theme, they'd also need to copy `/css/component/menu-dropdown.breakpoint.css` to their sub-theme too, otherwise this wouldn't work.